docs/en/modules/xpt2046.md

```diff
@@ -0,0 +1,129 @@
+# XPT2046 Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2016-04-14| [Starofall](https://github.com/Starofall) | [Frank Exoo](https://github.com/FrankX0) | [xpt2046.c](../../../app/modules/xpt2046.c)|
+
+XPT2046 is a touch controller used by several cheap displays - often in combination with the ILI9341 display controller.
+The module is build based on the libarys of [spapadim](https://github.com/spapadim/XPT2046/) and  [PaulStoffregen](https://github.com/PaulStoffregen/XPT2046_Touchscreen).
+
+
+## xpt2046.init(cs_pin, irq_pin, height, width)
```

Comment:
Here and below: Keep API doc consistent and don't list the parameters in the section title.